roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -99,15 +99,14 @@ policyConfig:
   bootstrapPolicyFile: {{ openshift_master_policy }}
   openshiftInfrastructureNamespace: openshift-infra
   openshiftSharedResourcesNamespace: openshift
-{# TODO: Allow users to override projectConfig items #}
 projectConfig:
   defaultNodeSelector: "{{ openshift.master.default_node_selector | default("") }}"
-  projectRequestMessage: ""
-  projectRequestTemplate: ""
+  projectRequestMessage: "{{ openshift.master.project_request_message | default("") }}"
+  projectRequestTemplate: "{{ openshift.master.project_request_template | default("") }}"
   securityAllocator:
-    mcsAllocatorRange: s0:/2
-    mcsLabelsPerProject: 5
-    uidAllocatorRange: 1000000000-1999999999/10000
+    mcsAllocatorRange: "{{ openshift.master.project_request_template | default("s0:/2") }}"
+    mcsLabelsPerProject: "{{ openshift.master.mcs_labels_per_project | default("5") }}"
+    uidAllocatorRange: "{{ openshift.master.uid_allocator_range | default("1000000000-1999999999/10000") }}"
```

Comment:
@spinolacastro My main goals with openshift_facts are the following:
1. A single place to set/update facts for the various components
2. Sane defaults for the majority of installations (taking into account cloud providers, etc)
3. Persisted "facts" between runs, so that if a user runs from different playbooks, or dynamic inventory, we do not overwrite their chosen data.

The only issue not currently addressed with this PR is item 1, but I know that openshift_facts is also a bit of a "beast", so feel free to tackle it, but if it becomes too much work, then we can file an issue and address it at a later time as well.